src/coverageLib/FileCoverage.ts

```diff
@@ -0,0 +1,368 @@
+import { Stmt, Expr } from "../parser";
```

Comment:
The reason I didn't name this `coverage` is because it's often included in the `.gitignore`. For example, if you run `jest --coverage` in a javascript project such as this one, it will create a `coverage/` directory, which you wouldn't want to commit. However, I could rename it to `coverageCollector` or something maybe?